MANIFESTO.md

```diff
@@ -0,0 +1,39 @@
+# Manifesto
+As Hitta.se's development team, we believe in:
+
+1. **Testing matters**
+   - Testing early and often exponentially decreases the amount of rework as compared to trying to find and fix bugs later in any lifecycle.
+   - That's why we test early and test often.
+   - We always dedicate time to expand our automated test suites by building both unit and end-to-end tests.
+2. **Quality above all**
+   - Quality takes priority over features and time.
+   - Quality has many facets, such as security, performance and even a great user experience.
+3. **No islands**
+   - We collaborate on design and specs.
+   - We make sure knowledge is shared.
+   - We do continuous code reviews.
+   - We commit and push frequently.
+   - We give honest feedback.
+   - Processes are a means to help individuals and interactions.
+   - Learn from others and help others learn from you.
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
```

Comment:
The comma should be a full stop and Don't over-architect a new sentence #grammar